documentation/modules/ref-kafka-rack.adoc

```diff
@@ -25,11 +25,11 @@ spec:
   kafka:
     # ...
     rack:
-      topologyKey: failure-domain.beta.kubernetes.io/zone
+      topologyKey: topology.kubernetes.io/zone
     # ...
 ----
 
-In the above example, the `failure-domain.beta.kubernetes.io/zone` node label will be used for scheduling Kafka broker Pods.
+In the above example, the `topology.kubernetes.io/zone` node label will be used for scheduling Kafka broker Pods.
```

Comment:
Better placed before the example?  `Above` might not be on same page in PDF. 

_In the following example, the `topology.kubernetes.io/zone` node label is used for scheduling Kafka broker Pods._